.travis.yml

```diff
@@ -33,6 +33,25 @@ matrix:
       script:
         - ./clang-format.sh
         - git diff --exit-code
+#adding code to run on power systems
+    - compiler: gcc
+      arch: ppc64le
+      env:
+        - CC="gcc-8" CXX="g++-8"
+
+    - compiler: clang
+      arch: ppc64le
+
+    - env:
+        - JOB_TYPE="clang-format"
+      arch: ppc64le  
+      addons:
+        apt:
+          packages:
+            - clang-format
+      script:
+        - ./clang-format.sh
+        - git diff --exit-code
```

Comment:
i don't think we need a second clang-format job. Code formatting shouldn't change across archs

Reply:
I think you missed this